Watchman.Tests/Lambda/LambdaAlarmTests.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon.CloudWatch;
+using Amazon.Lambda;
+using Amazon.Lambda.Model;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using Watchman.Configuration;
+using Watchman.Configuration.Generic;
+using Watchman.Engine;
+using Watchman.Engine.Generation;
+using Watchman.Tests.Fakes;
+using Watchman.Tests.IoC;
+
+namespace Watchman.Tests.Lambda
+{
+    public class LambdaAlarmTests
+    {
+        [Test]
+        public async Task AlarmCreatedWithCorrectProperties()
```

Comment:
This test is useful but doesn't seem to test the new stuff you added. Can we test that the evaluation period is configurable? something like copy the whole test and parameterise it with a period that goes into the config, removing most of the asserts?

Reply:
I've added additional tests.